docs/integrations/graphite.md

```diff
@@ -136,3 +136,23 @@ This will make the carbon ingestion emit logs for every step that is taking. *No
 - p99
 - p999
 - p9999
+
+## Querying
+
+M3 supports the the majority of [graphite query functions](https://graphite.readthedocs.io/en/latest/functions.html) and can be used to query metrics that were ingested via the ingestion pathway described above.
```

Comment:
Ehhh - so technically I think we support from version 0.9.10: https://graphite.readthedocs.io/en/0.9.10/functions.html
@jeromefroe - you know?